keps/sig-scheduling/3902-decoupled-taint-manager/README.md

```diff
@@ -173,7 +173,7 @@ Since there are no new API changes, we can skip Alpha and go to Beta directly.
 
 #### GA
 
-* Fix all reported bugs if any.
+* No negative feedback.
 
 ### Upgrade / Downgrade Strategy
```

Comment:
Please give a pass to the PRR Questionnaire to make sure no new questions were added since the last time this document was updated.

Reply:
No changed. it's up-to-date.